setup.py

```diff
@@ -0,0 +1,20 @@
+import setuptools
+
+setuptools.setup(
+    name="simGL", 
+    version="0.1",
+    author="Moisès Coll Macià, Graham Gower",
+    author_email="moicoll@birc.au.dk",
+    description="Simulate genotype likelihoods from haplotype genotype matrix.",
+    packages=["simGL"],
+    license = "ISC",
+    url = "https://github.com/RacimoLab/simGL",
+    install_requires=["numpy>=1.22","scipy>=1.7"],
```

Comment:
It's usually not necessary to specify a minimum version for the `install_requires` packages, unless you're explicitly using some feature that doesn't exist in older versions (e.g. the [new array interface in numpy](https://numpy.org/doc/stable/release/1.22.0-notes.html#implementation-of-the-nep-47-adopting-the-array-api-standard))